rfcs/text/0000-planning-for-the-dcp.md

```diff
@@ -0,0 +1,211 @@
+
+ All Scrum events are scheduled on the DCP calendar at a regular cadence. For example:
+![](../images/0000-scrum-calendar.png)
+
+### Refinement
+
+**Every two weeks**. There is a one hour meeting for **Product Owners**, **UXR**, and **Technical Leads** to collaborate on adding detail, owners, and order (but not estimates) to the items in the _dcp_ repo *Product Backlog* to reflect DCP priorities and address emerging issues. Dependencies between components are also clearly identified. (Refinement may also occur during the Sprint Review.)
+
+Questions or disagreements about priorities MUST be referred to the **Project Leads** on the [DCP PM Team mailing list](pm-team@data.humancellatlas.org) for resolution. They may resolve it in a manner of their own choosing, and respond to the original message on the DCP PM mailing list within two business days.
+
+***NOTE**: In the future, DCP roadmaps will be a source for a shared understanding of DCP priorities.*
+
+After this meeting, the **Product Owner** and **Technical Lead** for each component MUST ensure that the *Product Backlog* in their repo is refined and aligned to the DCP priorities.
+
+(See *Refinement* in [*Scrum Guide*](https://scrumguides.org/docs/scrumguide/v2017/2017-Scrum-Guide-US.pdf) and [*Nexus Guide*](http://scrumorg-website-prod.s3.amazonaws.com/drupal/2018-01/2018-Nexus-Guide-English_0.pdf) for guidance.)
+
+#### Refinement and ZenHub Pipelines
+
+New items in the *dcp* repo appear in the ZenHub *New* pipeline and MUST be triaged into the *Product Backlog* pipeline, the *Icebox* pipeline, or the *Closed* pipeline.
+
+The ZenHub *Product Backlog* pipeline is the top-level DCP product backlog which is ordered during Refinement.
+
+The ZenHub *Icebox* pipeline SHOULD be used with great constraint in the *dcp* repo. It's not a pipeline for avoiding hard decisions during Refinement. It's intended to document potential *futures* from the DCP roadmap. Individual component repo(s) MAY use the *Icebox* pipeline in a manner of their own choosing.
+
+No issues in the *dcp* repo MUST be assigned to the *Epic* pipeline. Individual component repo(s) MAY use the *Epic* pipeline in a manner of their own choosing.
+
+### Sprint Planning
+
+Prior to the start of the DCP Sprint, component teams MUST identify their *Sprint Backlog* - a slice of items from the the ZenHub *Product Backlog* pipeline in their repo which are assigned to the ZenHub Milestone for the Sprint and moved to the ZenHub *Sprint Backlog* pipeline.
+
+The number of items selected is solely up to the Development Team. 
+- [ZenHub story points](https://help.zenhub.com/support/solutions/articles/43000034282) MUST be assigned to all *Sprint Backlog* items. (Story points are required to enable [ZenHub burndown charts](https://help.zenhub.com/support/solutions/articles/43000010356-track-sprint-progress-with-burndown-charts) to track the progress of a Sprint.)
```

Comment:
I find it hard to see how story point estimates will be useful at this level since the levels of uncertainty around any estimates will be very high.

Reply:
What do you mean by _this level_? These are story points assigned to issues in your own repo for the next Sprint.

Reply:
I misunderstood the scope of your text above - I was thinking of a planning level above individual items but I see that's not what you are talking about. However, I'll make 2 points. 1) As above, emerging items would need the whole team to assess the story points, which isn't feasible in the time we have available and I'm not 100% sure of the value. And 2) even locally when we try to drill down our local story point estimations are usually very inaccurate. I believe this is partly due to inexperience estimating but mostly due to the complexity of the domain where our understanding of what we need to do is constantly changing and often only emerges when we try to do it.

Reply:
1) For the moment, we're **not** going to estimate story points for DCP-level issues. In the future, we may need rough t-shirt sizing for quarterly roadmaps but I'd defer that discussion to the future Roadmap RFC that the Project Leads are iterating on.
2) Use spikes to reduce complexity. Introspect during Daily Scrum / Team Retrospectives about why estimates are _off_.  We want to use this process to build the muscle - not shame anyone.